waspc/data/Generator/templates/server/package.json

```diff
@@ -7,6 +7,9 @@
   "scripts": {
     "start": "nodemon ./src/server.js",
     "debug": "DEBUG=server:* npm start",
+    "db-migrate-save": "prisma --experimental migrate save --schema=../db/schema.prisma",
```

Comment:
I added these two commands because they can be hard to write manually because schema is at weird place.